py-rse/automate.Rmd

```diff
@@ -304,7 +239,7 @@ knitr::include_graphics("figures/FIXME.png")
 
 As this diagram shows,
```

Comment:
It might be a good idea to add a sketch of the diagram (or maybe quickly make the figure in draw.io or something)? I'm assuming it's supposed to look something like the figures at the bottom of http://swcarpentry.github.io/make-novice/02-makefiles/index.html

Reply:
I added a rough sketch